spec/presenters/tree_builder_belongs_to_hac_spec.rb

```diff
@@ -0,0 +1,110 @@
+describe TreeBuilderBelongsToHac do
+  before do
```

Comment:
Dependencies are fine with `let`, ~~love~~ lazy-loading is always the answer :wink: